contractdata/PARIS/ESCALATION/XMAS/XMAS1.json

```diff
@@ -76,7 +80,12 @@
                                             },
                                             {
                                                 "$inarray": {
-                                                    "in": "$.LethalWeapons",
+                                                    "in": {
+                                                        "$or": [
```

Comment:
Does this work?

Reply:
Sorry, but maybe I used too much imagination and creativity here. You may see that all I want to do is to take Christmas explosives into consideration, so maybe there's a more acceptable type exist?

Reply:
You could just add the two repo ids of the explosives to the `LethalWeapons` array. I'm not entirely sure if explosive weapons can be checked like this however, so I suggest testing it.

Reply:
That makes sense... So the final question is, how to test? 🤯

Reply:
Play the contract and check if killing a target with your added explosives changes the counter?

Reply:
I mean how to replace the game file with my JSON

Reply:
You can either follow the steps [in the Readme.md under "Installation"](https://github.com/thepeacockproject/Peacock?tab=readme-ov-file#installation), and just run a dev version of Peacock with your modified files, or if you want you could copy the contract json into the "contracts" folder of your release version of Peacock, and then access the contract through "My contracts" in game.